library/src/main/java/com/cube/styleguide/adapter/viewholder/ColorItemViewHolder.kt

```diff
@@ -25,7 +25,9 @@ class ColorItemViewHolder(private val binding: ColorBannerViewBinding) : Recycle
 		list?.forEachIndexed { index, pair ->
 			val colorItemView = ColorItemViewBinding.inflate(LayoutInflater.from(itemView.context))
 			colorItemView.root.layoutParams = params
-			populateView(colorItemView, getMiddlePosition(list) == index, pair)
+            // If the colour list size is even it won't have a middle position, so instead we will take the index that is half of the size.
+            val isMiddle = if (list.size%2 == 0) list.size/2 else getMiddlePosition(list)
```

Comment:
It looks like the indents have changed here - can you fix this?

Reply:
Done :)